fuel/utils.py

```diff
@@ -87,3 +88,24 @@ def __getstate__(self):
 
         return cls
     return wrap_class
+
+
+def expand_axis_label(axis_label):
+    """Expand an abbreviated axis label.
+
+    Parameters
+    ----------
+    axis_label : str
+        Abbreviated axis label
+
+    """
+    if axis_label == 'b':
+        return 'batch'
+    elif axis_label == 'c':
+        return 'channel'
+    elif axis_label == 't':
+        return 'time'
+    elif re.match('^[0-9]+$', axis_label):
```

Comment:
Sure, 'w' and 'h' for images makes sense.
'f' for 'features' works as well, or we could just let it blank.